src/serlo-editor/plugins/sc-mc-exercise/renderer.tsx

```diff
@@ -77,7 +77,8 @@ export function ScMcExerciseRenderer({
                   />
                   <label
                     className={clsx(
-                      'serlo-spacing-fix flex cursor-pointer items-center [&>div>*]:ml-2',
+                      /* we override some styles in the answer content because it's not standalone */
```

Comment:
thanks for commenting this.
tbh. I don't really understand what you mean by 
> because it's not standalone

Reply:
improved comment